includes/admin.form.inc

```diff
@@ -145,7 +156,7 @@ function islandora_entities_settings($form, &$form_state) {
     ),
     '#description' => t('Number of Citations to show under Recent Citations/Citations Tab'),
     '#required' => TRUE,
-  );
+   );
```

Comment:
I think this is fixed now -- sorry I'm constantly fighting my code editor to not automatically do my spaces! @rosiel